k9mail/build.gradle

```diff
@@ -17,6 +17,27 @@ dependencies {
     compile 'com.android.support:support-v13:21.0.2'
     compile 'net.sourceforge.htmlcleaner:htmlcleaner:2.2'
     compile 'de.cketti.library.changelog:ckchangelog:1.2.1'
+
+    androidTestCompile ('com.jakewharton.espresso:espresso:1.1-r3' ) {
```

Comment:
would be easier to review if acceptance tests  / pgp related changes were separate PRs, not obvious if there are dependencies between the two